python/nav/portadmin/snmp/base.py

```diff
@@ -66,19 +65,14 @@ class SNMPHandler(object):
     # Port Access Control in a System.
     dot1xPaeSystemAuthControl = '1.0.8802.1.1.1.1.1.1.0'
 
-    netbox = None
-
     def __init__(self, netbox, **kwargs):
-        self.netbox = netbox
+        super().__init__(netbox, **kwargs)
         self.read_only_handle = None
         self.read_write_handle = None
         self.available_vlans = None
         self.timeout = kwargs.get('timeout', 3)
         self.retries = kwargs.get('retries', 3)
 
-    def __str__(self):
-        return self.netbox.type.vendor.id
-
```

Comment:
Keep this here, if you think this is only relevant to `SNMPHandler`. If this is generally only shown during development (live debugging, tests etc.) the thing that matters is consistency, but if this is shown to end users it's another matter. You might want to consider overriding `__repr__` to handle you dream situation for dev/debug.

Reply:
As mentioned above, I'm not sure why it would be relevant. If I cannot actually find it to be relevant, then I'd rather just remove it entirely.

Reply:
Really, I can't find any specific usage of a string representation of handlers in the code, and since the implementation doesn't appear to make much sense, I'm just going to remove it.

Reply:
Well, **durh**, the test suite found the one place this was being used: *By the test suite itself*, to determine the type of the class returned by the management factory.

Seems pretty braindead to me, fixed tests are coming up.